src/app.rb

```diff
@@ -0,0 +1,98 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './modules/list'
+require_relative './modules/check'
+
+class App
+  def initialize
+    @list_people = []
+    @list_books = []
+    @rentals = []
+  end
+
+  include ShowList
+  include CheckConditions
+
+  def create_person
+    puts 'Do you want to create a Student (1) or a Teacher (2) [Input the number]:'
+    create_person_input = gets.chomp.to_i
+    case create_person_input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Invalid please choose [1] or [2]'
+      create_person
+    end
+  end
+
+  def create_student
+    puts 'Age:'
+    student_age = gets.chomp.to_i
+    age_stud_checker(student_age)
+    student_age = gets.chomp.to_i
+
+    puts 'Name:'
+    student_name = gets.chomp.capitalize
+    label = 'Student'
+
+    puts 'Do you have parent permission? [y/n]:'
+    student_permission = gets.chomp
+    permission_checker(student_permission)
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix the `error` shown in the below `image` because I am unable to do the **"Create person"** part of the `Student parent permission` part 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/210277507-35556d11-5c24-41e8-bb5a-b1043aa5805a.png)

Reply:
thanks for noticing this @mukhammadrizooff, your review helps us learn about our mistakes and become better